src/PixiEditor/Models/Enums/DocumentTransformMode.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace PixiEditor.Models.Enums;
 internal enum DocumentTransformMode
 {
+    // Localization dependent on names!
```

Comment:
Maybe just add [Description("KEY")]?

Reply:
But I'm not localizing the name's of each value, I'm generating a key for each ActionDisplay